client/multi-currency-setup/tasks/store-settings-task/index.js

```diff
@@ -0,0 +1,204 @@
+		[ setIsAutomaticSwitchEnabledValue ]
+	);
+
+	const handleIsStorefrontSwitcherEnabledClick = useCallback(
+		( value ) => {
+			setIsStorefrontSwitcherEnabledValue( value );
+		},
+		[ setIsStorefrontSwitcherEnabledValue ]
+	);
+
+	const handleContinueClick = useCallback( () => {
+		setStatus( 'pending' );
+		submitStoreSettingsUpdate(
+			isAutomaticSwitchEnabledValue,
+			isStorefrontSwitcherEnabledValue
+		);
+		setStatus( 'resolved' );
+		setCompleted( true, 'setup-complete' );
+	}, [
+		setCompleted,
+		isAutomaticSwitchEnabledValue,
+		isStorefrontSwitcherEnabledValue,
+		setStatus,
+		submitStoreSettingsUpdate,
+	] );
+
+	return (
+		<WizardTaskItem
+			title={ interpolateComponents( {
+				mixedString: __(
+					'{{wrapper}}Review store settings{{/wrapper}}',
+					'woocommerce-payments'
+				),
+				components: {
+					wrapper: <span />,
+				},
+			} ) }
+			visibleDescription={ __(
+				'These settings can be changed any time by visiting the multi-currency settings',
+				'woocommerce-payments'
+			) }
+			index={ 2 }
+		>
+			<CollapsibleBody className="multi-currency-settings-task__body">
+				<p className="wcpay-wizard-task__description-element is-muted-color">
+					{ __(
+						'These settings can be changed any time by visiting the multi-currency settings',
+						'woocommerce-payments'
+					) }
+				</p>
+				<Card className="multi-currency-settings-task__wrapper">
+					<CardBody>
+						<CheckboxControl
+							checked={ isAutomaticSwitchEnabledValue }
+							onChange={ handleIsAutomaticSwitchEnabledClick }
+							data-testid={ 'enable_auto_currency' }
+							label={ __(
+								'Automatically switch customers to their local currency if it has been enabled',
+								'woocommerce-payments'
+							) }
+						/>
+						<div className="multi-currency-settings-task__description">
+							{ __(
+								'Customers will be notified via store alert banner.',
+								'woocommerce-payments'
+							) }
+						</div>
+						{ 'Storefront' === storeSettings.site_theme ? (
+							<>
+								<CheckboxControl
+									checked={ isStorefrontSwitcherEnabledValue }
+									onChange={
+										handleIsStorefrontSwitcherEnabledClick
+									}
+									data-testid={ 'enable_storefront_switcher' }
+									label={ __(
+										'Add a currency switcher to the cart widget',
+										'woocommerce-payments'
+									) }
+								/>
+								<br />
+							</>
+						) : null }
+						<div className="wcpay-wizard-task__description-element is-muted-color">
+							{ __(
+								'A currency switcher is also availabie in your widgets.',
+								'woocommerce-payments'
+							) }
+						</div>
+					</CardBody>
+				</Card>
+				<Button
+					isBusy={ 'pending' === status }
+					disabled={ 'pending' === status }
+					onClick={ handleContinueClick }
+					isPrimary
+				>
+					{ __( 'Continue', 'woocommerce-payments' ) }
+				</Button>
+				<Button
+					isBusy={ 'pending' === status }
+					disabled={ 'pending' === status }
```

Comment:
Looking at the use of `status` maybe you can rename it to something like `isPending` and use a boolean instead. This way you can get rid of 4 comparisons.

Reply:
Done in 73cd066.